src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/ServiceFilesMediatorTest.kt

```diff
@@ -0,0 +1,199 @@
+/**
+Copyright 2018 BarD Software s.r.o
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+package com.bardsoftware.papeeria.backend.cosmas
+
+import com.google.cloud.storage.Blob
+import com.google.cloud.storage.Storage
+import com.google.cloud.storage.StorageException
+import com.google.cloud.storage.contrib.nio.testing.LocalStorageHelper
+import com.google.protobuf.ByteString
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito
+import java.io.IOException
+import java.time.Clock
+
+/**
+ * Some tests for ServiceFilesMediator
+ *
+ * @author Aleksandr Fedotov (iisuslik43)
+ */
+class ServiceFilesMediatorTest {
+    private val BUCKET_NAME = "papeeria-interns-cosmas"
+    private var service = getServiceForTests()
+    private var mediator = getMediatorForTests()
+
+    @Before
+    fun testInitialization() {
+        mediator = getMediatorForTests()
+        println()
```

Comment:
?

Reply:
like in `CosmasGoogleCloudServiceTest`, it splittes log from different tests